GenderPayGap.Database/GpgDatabaseContextFactory.cs

```diff
@@ -1,4 +1,6 @@
-﻿using GenderPayGap.Core;
+﻿using System;
+using GenderPayGap.Core;
+using GenderPayGap.Extensions.AspNetCore;

```

Comment:
Super nitpicky, but we don't need these imports now we're not changing anything else in this file.